README.md

```diff
@@ -587,6 +590,39 @@ This output can be then passed to another step in json format:
 foreach: '{{.step.prefixStrings.children | toJson}}'
 ```
 
+#### Resources <a name="resources"></a>
```

Comment:
What is the link used for ?

Reply:
It's used line 359 to give a quick link to this section.